src/arrayMethodReduce.js

```diff
@@ -5,7 +5,23 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator;
```

Comment:
Let's write down the initial value here
```suggestion
    let accumulator = initialValue;
```